bdk_chain/tests/test_keychain_txout_index.rs

```diff
@@ -56,14 +60,9 @@ fn test_store_all_up_to() {
 
 #[test]
 fn test_pad_all_with_unused() {
-    let mut txout_index = init_txout_index();
+    let (mut txout_index, external_desc, _) = init_txout_index();
 
-    let external_spk3 = txout_index
-        .keychains()
-        .get(&TestKeychain::External)
-        .unwrap()
-        .at_derivation_index(3)
-        .script_pubkey();
```

Comment:
IMO this code was useful.. It shows how to get the keychains from a tracker..

Reply:
But this is a test, not an example